filter2.js

```diff
@@ -4,3 +4,6 @@ var misc = [3, "true", {a: 1, b: 2}, 7, [1, 2, 3], ['a', 'b', 'c'], "my favorite
 // Don't worry about strings in the nested array (misc[5] is considered an array, not a string)
 
 // Your code goes here
+var stringFilter= misc.filter(misc=>typeof misc ==='string')
+
+console.log(stringFilter);
```

Comment:
you set this up really well! however, you filtered out everything that **isn't** a string, rather than everything that **is** a string. 😆 That said, I trust you could address this without too much trouble